src/test/java/org/jilt/test/data/defaultvalue/DefaultValueWithLombok.java

```diff
@@ -0,0 +1,39 @@
+package org.jilt.test.data.defaultvalue;
+
+import org.jilt.Builder;
+
+import java.util.Set;
+
+@Builder
+public class DefaultValueWithLombok {
+
+    @lombok.Builder.Default
+    public int attr1 = 1;
+
+    @lombok.Builder.Default
+    public String attr2 = "attr2";
+
+    @lombok.Builder.Default
+    public boolean attr3 = true;
+
+    public int attr4 = 1000;
+
+    @lombok.Builder.Default
+    public int attr5 = 1000;
```

Comment:
Let's try referencing the field here, to check if that works? So something like

```suggestion
    public int attr5 = attr1 + 1000;
```